.clang-format

```diff
@@ -0,0 +1,13 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+
+ColumnLimit: 120
```

Comment:
We've more-or-less gone back to 80 in our python code, should this still be 120?

Reply:
I'm fine with 80, but then again I haven't written c++ in a long time..

Reply:
Ok, given that I think most folks will implement this as a commit hook or similar, I think it makes sense to keep it at 120 to allow users to occasionally exceed it (as we do in python). But I'll just add in writing that keeping it to 80 is preferred.